clicache/src/impl/PdxLocalReader.cpp

```diff
@@ -308,6 +314,16 @@ namespace Apache
           return m_dataInput->ReadArrayOfByteArrays();
         }
 
+				array<array<SByte>^>^ PdxLocalReader::ReadArrayOfSByteArrays(String^ fieldName )
```

Comment:
Fixed. I'm not sure, but what may have happened is that I had turned off clang-format on my machine due to it crashing during the build of clicache. So perhaps some of these formatting issues were prexistent but didn't show up since they hadn't been changed in a long time.